packages/cli/package.json

```diff
@@ -160,13 +160,13 @@
   },
   "repository": "coveo/cli",
   "scripts": {
-    "postpack": "rimraf oclif.manifest.json",
-    "prepack": "rimraf lib && npm run build && oclif-dev manifest && oclif-dev readme",
-    "build": "tsc -b",
+    "snyk:prepare": "cp ../../package-lock.json ./package-lock.json && npm i",
```

Comment:
I was looking at https://github.com/louis-bompart/literate-bassoon to understand why Snyk is broken, but the link is dead.

I guess we need to copy the top level package-lock to the "local" package because Snyk is unable to scan otherwise ?

Does that mean that snyk is looking at all dependencies for all packages, and so will report N time the same problem, where N is the amount of packages we have ?

Reply:
Sorry, forgot to put the repo public, done. 🤦 

So yeah, tldr snyk does not support npm workspace. A feature request has been opened. (It does not support lerna too btw)

> I guess we need to copy the top level package-lock to the "local" package because Snyk is unable to scan otherwise ?
I copy the package-lock so I could provide some insight of what is the version to use to npm (it's not a silver bullet tho)

> Does that mean that snyk is looking at all dependencies for all packages, and so will report N time the same problem, where N is the amount of packages we have ?
Well that was the behaviour I hoped for, but in essence, because Snyk does not scan the workspaces (a.k.a. packages) packages from the root, despite them being here :(
So no, for 1 vulnerability in a dependency, we will have the problem reported only in the package that use the dependency.